include/deal.II/grid/grid_generator.h

```diff
@@ -304,6 +304,49 @@ namespace GridGenerator
   cheese(Triangulation<dim, spacedim> &   tria,
          const std::vector<unsigned int> &holes);
 
+  /**
+   * \brief Plate with a hole.
+   *
+   * Generate a plate with a hole. The geometry consists of 2 parts:
+   * First, a square part with size @p outer_radius and a hole of radius @p inner_radius .
+   * Cells in this part will have TransfiniteInterpolationManifold with id
+   * @p tfi_manifold_id attached. Additionally, faces at the hole will use
+   * PolarManifold (in 2D) or CylindricalManifold (in 3D). The center of this
+   * part can be prescribed via @p center .
+   * Second, the remaning bulk part which is specified via padding
+   * parameters @p pad_bottom, @p padding_top, @p padding_left and @padding_right.
+   * All cells in this part will have FlatManifold attached.
+   *
+   * Here is the grid (after one global refinement) in 2D, collored according to
```

Comment:
the default constructed grid

Reply:
it's not default, but I will say `non-symmetric`. I wanted to give an image where all paddings are different.